.gitignore

```diff
@@ -33,3 +33,6 @@
 
 public/sitemap.xml
 public/system/
+
+# Ignore vim swap files
```

Comment:
Are you using Vim as your editor? If so, could you please exclude this from CONSUL's `.gitignore` and add it to your `.gitignore_global`? We try to keep said file as clean as possible and we only modify it to exclude CONSUL-specific files

Reply:
@aitbw Done!